### PR TITLE
Add Sahilsaini99's pull

### PR DESCRIPTION
**Pull request by iniaslihas**
check out the Commits 101 drop-down.
through this I have learnt how to created a branch, to created a file and have made a commit, and opened a pull request. I have learnt many new things by this introduction.

### CODE REVIEW

File: _posts/0000-01-02-Sahilsaini99.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```
Comment (Sahilsaini99) [inline _posts/0000-01-02-Sahilsaini99.md]: *are you welcomed*

Comment (Sahilsaini99) [reply _posts/0000-01-02-Sahilsaini99.md]: **you are directed to next slide**

Comment (Sahilsaini99) [inline _posts/0000-01-02-Sahilsaini99.md]: *can you move*

**Review by iniaslihas — COMMENTED** [review]
*well Done*

**Pull request merged**